File: SoftUni - C# OOP/Polymorphism/Polymorphism - Lab/02. Animals/Cat.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Animals
{
    public class Cat : Animal
    {
        public Cat(string name, string food) : base(name, food)
        {
        }
        public override string ExplainSelf()
        {
            return $"I am {this.Name} and my fovourite food is {this.Food} {Environment.NewLine}MEEOW";         
        }
    }
}

File: SoftUni - Java OOP/Polymorphism/Polymorphism - Lab/_03_Animals/Cat.java
package _03_Animals;

public class Cat extends Animal{
    public Cat(String name, String favouriteFood) {
        super(name, favouriteFood);
    }

    @Override
    public String explainSelf() {
        return "%s\nMEEOW".formatted(super.explainSelf());
    }
}
